src/topp/IDFileConverter.cpp

```diff
@@ -615,28 +617,49 @@ class TOPPIDFileConverter :
     else if (out_type == FileTypes::FASTA)
     {
       Size count = 0;
-      ofstream fasta(out.c_str(), ios::out);
+
+      bool concat = getFlag_("concatenate_peptides");
+      //To avoid creating (K/R)P sites and losing information when the FASTA entry is used by
```

Comment:
[KP]|P sites are created in any case. The question is just how this affects the preceeding peptides if a peptide starts with P.